src/rad/resources/schemas/wfi_image-1.0.0.yaml

```
@@ -16,6 +16,11 @@ properties:
       - $ref: common-1.0.0
       - type: object
         properties:
+          wcs:
+            title: WCS object
+            anyOf:
+              - tag: tag:stsci.edu:gwcs/wcs-1.0.0
```

@nden Here's the PR that added tag 'wildcard' support:
https://github.com/asdf-format/asdf/pull/858

I am not immediately finding usage of this within rad (I was hoping to point to an existing example).

If a wildcard seems worthwhile here one option would be to restrict it to only "wcs-1.x.x" versions (in case there is a "2.0.0" that breaks things.
```suggestion
              - tag: tag:stsci.edu:gwcs/wcs-1.*
```
The more liberal option would be to use "wcs-*". I did not test either of these locally so I'm not sure if there are other issues that might be caused by the use of a wildcard here.

---

Thanks !This works and regression tests now pass. Reading files with various WCS versions works as well.